python/tests/integration/arcticdb/version_store/test_categorical.py

```diff
@@ -81,19 +84,40 @@ def test_categorical_empty(lmdb_version_store, sym):
     lib = lmdb_version_store
     lib.write(sym, df)
     read_df = lib.read(sym).data
+    # In Pandas 1.0, an Index is used by default for any an empty dataframe or series is created,
+    # except if there are categorical columns in which case a RangeIndex is used.
+    #
+    # In Pandas 2.0, RangeIndex is used by default for _any_ an empty dataframe or series is created.
+    # See: https://github.com/pandas-dev/pandas/issues/49572
+    assert isinstance(df.index, pd.RangeIndex)
+    assert isinstance(read_df.index, pd.RangeIndex)
     assert_frame_equal(df, read_df)
 
 
 def test_categorical_with_integers(lmdb_version_store, sym):
     c = pd.Categorical(np.arange(6))
-    df = pd.DataFrame({"int": np.arange(6), "cat": c})
+    df = pd.DataFrame({"int": np.arange(6), "cat_int": c})
     lib = lmdb_version_store
     lib.write(sym, df)
     read_df = lib.read(sym).data
     # Not pickled
     assert lib.get_info(sym)["type"] == "pandasdf"
     # should be category
-    assert read_df.cat.dtype == "category"
+    assert read_df.cat_int.dtype == "category"
+    if IS_PANDAS_TWO and (sys.maxsize <= 2**32 or sys.platform.startswith("win32")):
```

Comment:
We do not support 32-bit platforms anyway?

Reply:
Removed in 5a698e2.